src/bootstrap/src/core/config/config.rs

```diff
@@ -2091,6 +2096,8 @@ impl Config {
             );
         }
 
+        config.rust_std_features = std_features.unwrap_or_default();
```

Comment:
@onur-ozkan does `unwrap_or_default` work here? or do we need to explicitly provided a default value with `unwrap_or($default)`

Reply:
We need to include `panic_unwind` feature by default.

Reply:
> we are including it in config

Do you mean `std_features` function ? I think defaults should be handled here.